### PR TITLE
Fix frequency plot

### PR DESCRIPTION
**Pull request by cosa65**
# Description

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-1812

#### Link to staging deployment URL (or set N/A)
https://ui-martinfosco-ui687.scp-staging.biomage.net/

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

**Comment by aerlaut** [comment]
Have you checked if this issue happens in other plots as well?

File: src/__test__/components/plots/FrequencyPlot.test.jsx [hunk 1]
```diff
@@ -39,8 +41,11 @@ describe('Frequency plot tests', () => {
     fetchMock.mockIf(/.*/, mockAPI(generateDefaultMockAPIResponses(fake.EXPERIMENT_ID)));
   });
 
-  it('Updates CSV data on render', () => {
-    renderFrequencyPlot();
+  it('Updates CSV data on render', async () => {
+    await act(async () => {
+      renderFrequencyPlot();
+    });
```
Comment (aerlaut) [inline src/__test__/components/plots/FrequencyPlot.test.jsx]: Do we need this? I think I can see that UI changes inside render is wrapped in `act()`: https://github.com/testing-library/react-testing-library/blob/main/src/pure.js#L109-L115

Comment (cosa65) [reply src/__test__/components/plots/FrequencyPlot.test.jsx]: Yes, it fails if not because it doesn't wait for all of the redux actions to go through

Comment (aerlaut) [reply src/__test__/components/plots/FrequencyPlot.test.jsx]: Oh ok. Can you move this `await act()` inside `renderFrequencyPlot`? That way this await doesn't have to be repeated for all calls to `renderFrequencyPlot`.

**Review by aerlaut — APPROVED** [review]

**Pull request merged**